src/tests/suite/daos_test.c

```diff
@@ -378,7 +378,7 @@ main(int argc, char **argv)
 	memset(tests, 0, sizeof(tests));
 
 	while ((opt = getopt_long(argc, argv,
-				  "ampcCdtTVizxADKeoROg:n:s:u:E:f:Fw:W:hrNvbSXl:",
+				  "ampcCdtTVizUxADKeoROg:n:s:u:E:f:Fw:W:hrNvbSXl:",
```

Comment:
Is L336 correct, that seems to say --dedup is lowercase u (which is actually used twice).

Reply:
You're right. I'll fix it.